src/data/list/basic.lean

```diff
@@ -2540,7 +2540,7 @@ end
 @[to_additive]
 lemma head_mul_tail_prod' [monoid α] (L : list α) :
   (L.nth 0).get_or_else 1 * L.tail.prod = L.prod :=
-by { cases L, { simp, refl, }, { simp, }, }
+by { cases L; simp }
```

Comment:
```suggestion
by cases L; simp
```
Why not go all the way and get rid also of the braces?  (-;

Reply:
Good idea!